### ISSUE TITLE
embedded tweet is not working properly

### ISSUE THREAD

**Issue by jayne-mast** [smashingmagazine/redesign#251]
**- Do you want to request a *feature* or report a *bug*?**
report a bug

**- What is the current behavior?**
Embedded tweets in articles don't show up properly in Firefox 57 on macOS High Sierra

**- If the current behavior is a bug, please provide the steps to reproduce.**
- Open Firefox 57 
- Go to https://www.smashingmagazine.com/2017/11/from-cats-with-love-welcome-the-new-smashing-membership/#so-what-is-smashing-membership-
- notice the tweet is not visible:
<img width="747" alt="screen shot 2017-11-22 at 09 02 45" src="https://user-images.githubusercontent.com/1258335/33116240-57e08764-cf64-11e7-919b-90127f885940.png">

**- What is the expected behavior?**
The content of the tweet is visible

**- Please mention your operating system version and the version of your browser**
Firefox 57 on macOS High Sierra 10.13.1

**Comment by sandrarobotos** [comment]
I am also seeing this: Chrome 62 on macbook pro 15inch retina, macOS Sierra 10.12.6.

If I shrink the browser window, to 900px for example, the pullquotes show; if I refresh the page at this size the pullquotes animate properly as I scroll down the page.

Hope this helps! 🙂

**Comment by vitalyfriedman** [comment]
Wow, that's really weird. @pverbeek @sandrarobotos Could you be very specific at which resolution they start disappearing?..

**Comment by jayne-mast** [comment]
hmm… I can't reproduce it anymore. @sandrarobotos do you still see it happening?

**Comment by sandrarobotos** [comment]
@pverbeek @vitalyfriedman, yes I still see this happening. Screen resolution 1440x900. I start seeing this at 1300px. Smaller screen sizes are fine, animating in and everything. 
I'm on a laptop right now, so can't tell if this happens above 1440px.
